test/e2e/network/networking.go

```diff
@@ -206,7 +206,9 @@ var _ = SIGDescribe("Networking", func() {
 			}
 		})
 
+		// skip because of #95565
```

Comment:
maybe clarify a _little_ more?

    // This ought to work but for some reason it doesn't. http://issues.k8s.io/95565

Reply:
yep